reference.rst

```diff
@@ -16,3 +16,4 @@ Reference
    reference/extensions
    reference/environment
    reference/conan_server
+   reference/backup_sources
```

Comment:
Doesn't feel the best place for this

Reply:
Yep, I also know we talked about this. I've moved it to a new "Other" section which I _think_ was the call we made, but let me know otherwise :)